content/contributing/documentation/rst_cheat_sheet.rst

```diff
@@ -216,18 +216,26 @@ Use the `command` markup to highlight a command.
 Icons
 -----
 
-Use the `icon` markup to add a class name of an icon. The icon set used is *Font Awesome*. It is
-recommended to accompany an icon with a :ref:`contributing/markups/guilabel` as a descriptor,
-however, it is not mandatory.
+Use the `icon` markup to add a class name of an icon. There are two icon sets used in the Odoo user
+interface: `FontAwesome4 <https://fontawesome.com/v4/icons/>`_ and :doc:`Odoo UI
+<../../developer/reference/user_interface/icons>`. It is recommended to accompany an icon with a
```

Comment:
Nitpicking: if you have to go back all the way up to the root, you could as well start from the root:
```suggestion
</developer/reference/user_interface/icons>`. It is recommended to accompany an icon with a
```